### PR TITLE
Map Keras LocallyConnected1D

### PR DESCRIPTION
**Pull request by hkvision**
## What changes were proposed in this pull request?

locallyconnected1d

## How was this patch tested?

Jenkins

### CODE REVIEW

File: pyspark/bigdl/keras/converter.py [hunk 1]
```diff
@@ -1614,7 +1621,7 @@ def create_highway(self):
         else:
             activation = get_activation_by_name(self.config["activation"],
                                                 "%s_%s" % (self.config["name"], self.config["activation"]))
-        blayer = BLayer.Highway(size=self.input_shape[1],
+        blayer = BLayer.Highway(size=int(self.input_shape[1]),
```
Comment (zhichao-li) [inline pyspark/bigdl/keras/converter.py]: is this a must that surround with int?

File: pyspark/bigdl/keras/converter.py [hunk 2]
```diff
@@ -1190,7 +1197,7 @@ def create_atrousconvolution1d(self):
         if not self.config["bias"]:
             raise Exception("Only bias=True is supported for AtrousConvolution1D")
 
-        h = self.input_shape[2]
+        h = int(self.input_shape[1])
```
Comment (zhichao-li) [inline pyspark/bigdl/keras/converter.py]: add a dedicated unitest for this which would fail if we use `self.input_shape[2]`

**Comment by zhichao-li** [comment]
LGTM pending on adding unit test.

File: pyspark/bigdl/keras/converter.py (hunk 1 above)
Comment (hkvision) [reply pyspark/bigdl/keras/converter.py]: I don't know. But you surround everything related to shape with int and thus I do this for everything to be consistent.

**Comment by hkvision** [comment]
jenkins passed http://172.168.2.101:8080/job/BigDL-PR-V/652/

**Pull request merged**